Core/Graphics/Source/DeviceImpl.cpp

```diff
@@ -49,6 +50,11 @@ namespace Babylon::Graphics
         DisableRendering();
     }
 
+    uintptr_t DeviceImpl::GetId()
+    {
+       return g_bgfxId;
```

Comment:
Is this right? What if someone creates another device while an existing device is still alive? Shouldn't this be stored as a member of DeviceImpl?

Reply:
I don't think so, because this is a static method. Also, bgfx would not allow another device to be created while the first one is alive, bgfx only supports the usage of a single graphics device at each time.